x-pack/plugins/reporting/public/management/index.ts

```diff
@@ -15,7 +15,7 @@ import type { SharePluginSetup } from '../shared_imports';
 export interface ListingProps {
   apiClient: ReportingAPIClient;
   capabilities: ApplicationStart['capabilities'];
-  license$: LicensingPluginSetup['license$']; // FIXME: license$ is deprecated
```

Comment:
Oops, this fix should have been in https://github.com/elastic/kibana/pull/122482